packages/ra-ui-materialui/src/theme/houseTheme.ts

```diff
@@ -0,0 +1,184 @@
+import {
+    alpha,
+    createTheme,
+    darken,
+    Theme,
+    PaletteOptions,
+} from '@mui/material';
+import { RaThemeOptions } from './types';
+
+/**
+ * House: A young and joyful theme.
+ *
+ * Uses rounded corners, blurry backdrop, large padding, and a bright color palette.
+ */
+
+const componentsOverrides = (theme: Theme) => ({
+    MuiBackdrop: {
+        styleOverrides: {
+            root: {
+                backgroundColor: alpha(darken('#000C57', 0.4), 0.2),
+                backdropFilter: 'blur(2px)',
+                '&.MuiBackdrop-invisible': {
+                    backgroundColor: 'transparent',
+                    backdropFilter: 'blur(2px)',
+                },
+            },
+        },
+    },
+    MuiOutlinedInput: {
+        styleOverrides: {
+            input: {
+                padding: `${theme.spacing(1)} ${theme.spacing(2)}`,
+            },
+        },
+    },
+    MuiTab: {
+        styleOverrides: {
+            root: {
+                padding: 0,
+                height: 38,
+                minHeight: 38,
+                borderRadius: 6,
+                transition: 'color .2s',
+
+                '&.MuiButtonBase-root': {
+                    minWidth: 'auto',
+                    paddingLeft: 20,
+                    paddingRight: 20,
+                    marginRight: 4,
+                },
+                '&.Mui-selected, &.Mui-selected:hover': {
+                    color: theme.palette.primary.contrastText,
+                    zIndex: 5,
+                },
+                '&:hover': {
+                    color: theme.palette.primary.main,
+                },
+            },
+        },
+    },
+    MuiTableRow: {
+        styleOverrides: {
+            root: {
+                '&:last-child td': { border: 0 },
+            },
+        },
+    },
+    MuiTableCell: {
+        styleOverrides: {
+            root: {
+                padding: theme.spacing(2),
+                '&.MuiTableCell-sizeSmall': {
+                    padding: theme.spacing(1.5),
+                },
+                '&.MuiTableCell-paddingNone': {
+                    padding: theme.spacing(0.5),
+                },
+            },
+        },
+    },
+    MuiTabs: {
+        styleOverrides: {
+            root: {
+                height: 38,
+                minHeight: 38,
+                overflow: 'visible',
+            },
+            indicator: {
+                height: 38,
+                minHeight: 38,
+                borderRadius: 6,
+                border: `1px solid ${theme.palette.primary.light}`,
+                boxShadow: theme.shadows[1],
+            },
+            scrollableX: {
+                overflow: 'visible !important',
+            },
+        },
+    },
+    MuiTextField: {
+        defaultProps: {
+            variant: 'outlined' as const,
+        },
+    },
```

Comment:
Add margin between glued input

```suggestion
    },
    MuiFormControl: {
        defaultProps: {
            margin: 'normal' as const,
        },
    },
```

Reply:
Fixed in 0bc94a4a2f126a057677e5c312eb3f97abc02bb3, thanks